hack/clean-testing-github-repos

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+import os
+import re
+
+import github
+
+repo_name_re = r"release-bot-test-\w{16}"
+
+
+def main():
+    # This token needs a scope to delete repositories, I suggest creating a new one:
```

Comment:
Maybe is better to use 'We' instead of 'I'

Reply:
I actually don't agree with you but I'll try to reword it even better.